app/src/main/res/layout/fragment_forms.xml

```diff
@@ -76,29 +75,31 @@
 
         <ImageView
             android:id="@+id/syncIconSuccess"
-            android:layout_width="@dimen/small_icon_size"
-            android:layout_height="@dimen/small_icon_size"
+            android:layout_width="@dimen/medium_icon_size"
+            android:layout_height="@dimen/medium_icon_size"
+            android:layout_marginEnd="@dimen/small_margin"
             android:contentDescription="@null"
             app:layout_constraintBottom_toBottomOf="@id/syncInfoSuccess"
             app:layout_constraintEnd_toStartOf="@id/syncInfoSuccess"
             app:layout_constraintHorizontal_chainStyle="packed"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="@id/syncInfoSuccess"
+            app:layout_constraintTop_toBottomOf="@id/formsList"
```

Comment:
This constraint doesn't do anything because we have the top constraint already defined one line above(app:layout_constraintTop_toTopOf="@id/syncInfoSuccess"). If you want the same layout as for the sync/text above then we need to move this constraint to the syncInfoSuccess TextView(the syncInfo TextView also has a top margin: android:layout_marginTop="@dimen/small_margin" if you want to add it as well).

Reply:
yep, I think we can remove this one :)